src/Components/Todo/TodoItem.tsx

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  onTodoRemove: (todoId: number) => void;
+  isRemovingCompleted: boolean;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  onTodoRemove,
+  isRemovingCompleted,
+}) => {
+  const [isRemoving, setIsRemoving] = useState<boolean>(false);
+  const isNewTodo = todo.id === 0;
+
+  const isRemovingCompletedTodo = isRemovingCompleted && todo.completed;
+  const isLoading = isNewTodo || isRemoving || isRemovingCompletedTodo;
+
+  const handleRemoveTodo = () => {
+    onTodoRemove(todo.id);
+    setIsRemoving(true);
+  };
+
+  return (
+    <div
+      className={classNames(
+        'todo',
+        { completed: todo.completed },
+      )}
+    >
+      <label className="todo__status-label">
+        <input
+          type="checkbox"
```

Comment:
The input element for the checkbox is missing an onChange handler to toggle the todo's completion status.

Reply:
should this function be implemented in this task?